src/pages/HomePage/HomePage.tsx

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
+import React, { CSSProperties, useEffect } from 'react';
+import { motion, Variants } from 'framer-motion';
+import clsx from 'clsx';
 
-import { useTranslation } from 'react-i18next';
+import BackgroundImage from 'assets/images/background.png';
 
-const HomePage: React.FC<IProps> = ({}) => {
-  const { t } = useTranslation();
+import RoutesList from './items/RoutesList';
 
-  return <div>{t('common:test')}</div>;
+import { RoutesMap } from './items/RoutesList/RoutesMap';
+import ROUTES from 'src/ROUTES';
+
+import './home-page.scss';
+
+const HomePage: React.FC<IProps> = ({ className }) => {
+  useEffect(() => {
+    document.title = ROUTES.home.title || 'SpaceXPlorer';
```

Comment:
SpaceXplorer powinien być w CONSTANTSach
```suggestion
    import { APP_TITLE } from 'CONSTANS'
    ...
    document.title = ROUTES.home.title || APP_TITLE;
```